packages/block-library/src/image/editor.scss

```diff
@@ -1,39 +1,13 @@
 // Provide special styling for the placeholder.
 // @todo this particular minimal style of placeholder could be componentized further.
 .wp-block-image.wp-block-image {
-
-	// Show Placeholder style on-select.
-	&.is-selected .block-editor-media-placeholder {
-		// Block UI appearance.
-		color: $gray-900;
-		background-color: $white;
-		box-shadow: inset 0 0 0 $border-width $gray-900;
-		border: none;
-
-		// Disable any duotone filter applied in the selected state.
-		filter: none !important;
-
-		// @todo this should eventually be overridden by a custom border-radius set in the inspector.
-		border-radius: $radius-block-ui;
-
-		> svg {
-			opacity: 0;
-		}
-
-		.components-placeholder__illustration {
-			display: none;
-		}
-
-		&::before {
-			opacity: 0;
-		}
+	.block-editor-media-placeholder.is-small {
+		min-height: 60px;
 	}
+
 	.block-bindings-media-placeholder-message {
 		opacity: 0;
 	}
```

Comment:
We also need to remove this CSS, otherwise the message will not be visible when the source is bound:

![image](https://github.com/user-attachments/assets/42654127-91a4-4da7-86ae-47983a04342a)